tool/gravity/cli/update_test.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"path/filepath"
+	"testing"
+
+	apptest "github.com/gravitational/gravity/lib/app/service/test"
+	"github.com/gravitational/gravity/lib/blob/fs"
+	"github.com/gravitational/gravity/lib/defaults"
+	"github.com/gravitational/gravity/lib/loc"
+	"github.com/gravitational/gravity/lib/localenv"
+	"github.com/gravitational/gravity/lib/ops/opsservice"
+	"github.com/gravitational/gravity/lib/storage/keyval"
+
+	"gopkg.in/check.v1"
+)
+
+func TestCLI(t *testing.T) { check.TestingT(t) }
```

Comment:
Are tests in tool/ executed as a part of "make test"?

Reply:
Yes, but I just noticed that we don't run enterprise package tests - something to fix in future.